docs/build/quickstarts.md

```diff
@@ -1,3 +1,200 @@
+
+```yaml
+services:
+  myService:
+    get: src/services/myService
+workers:
+  defaultWorker:
+    services: [ myService ]
+    spells: []
+deals:
+  defaultWorker:
+    minWorkers: 1
+    targetWorkers: 3
+```
+
+### Deal deploy
+
+`fluence deal deploy` will deploy all deals defined in `fluence.yml`.
+
+See `fluence deal deploy --help` for information on more settings to `deal deploy` command.
+
+As deployment happens through blockchain, you will be prompted to open a URL to sign Transaction with your Metamask wallet.
+
+```
+To approve transactions to your wallet using metamask, open the following url:
+
+https://cli-connector.fluence.dev/?wc=000000000000000000000%402&relay-protocol=irn&symKey=000000000000000
+```
+
+Open it, and connect Metamask. After a short while, you will see a Metamask prompt to sign Transaction.
+
+TODO: METAMASK_PROMPT_SCREENSHOT.JPG
+
+After a short while, you will see a message like this one:
+
+```
+# Waiting for transaction to be mined......
+
+Success!
+
+created deals:
+  defaultWorker:
+    deal: https://mumbai.polygonscan.com/address/0x00000000000000000000000000000000
+    worker definition: bafkreigu3u3swrzq4tingjbp77ozozacbcsujm3wnnr7x4xbeliwr5zkvi
+    timestamp: 2023-08-31T18:07:01.122Z
+```
+
+That means your Deal was deployed succesfully. After a short while, Providers will catch up and deploy it.
+
+### Call a function over the subnet
+
+In your freshly-initialized project, there's a file `src/aqua/main.aqua`. Among others, it contains a function `runDeployedServices` which resolves a Subnet, and calls a function on each of the Workers in the Subnet.
+
+To run it, simply run the following command:
+```bash
+fluence run -f 'runDeployedServices()'
+```
+
+You will see a list of Answers printed. Some of them may contain no answer, and `worker_id` will be null. That means that this specific Compute Peer has not deployed the Deal yet. Try again in a while, and it should appear.
+
+### Inspect deployed deal
+```bash
+fluence run -f 'showSubnet()'`
+```
+
+```json
+[
+    {
+        "services": [
+            "myService",
+            "worker-spell"
+        ],
+        "worker_id": "12D3KooLANJSDNdoandjqlwkDNIBNnnao12nWNj0uaJIKSALn"
+    },
+    {
+        "services": [
+            "myService",
+            "worker-spell"
+        ],
+        "worker_id": "12D3KooWKGU8FFyw5Ek7wWREX2KKEs2AjxnaojsndoJOWNSJd"
+    },
+    {
+        "services": [
+            "myService",
+            "worker-spell"
+        ],
+        "worker_id": "12D3KoonONmakW291NlajwyrnbLSAMNpqNAO21NQJWJSnsnwo"
+    }
+]
+```
+
+### Update deal
+
+Now, if you want to change something in one of your Services or Spells, you can do that, and simply call `deal deploy` again will update existing Deal in place.
```

Comment:
this is a big deal, imo and should be in neon flashing bold: deals are mutable and fluence cli manages the on-ch update process (for additional gas).

Reply:
Not sure how to communicate its importance here.

Reply:
i guess it could/should be done when/where deals a re covered.